_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
+Although it's already GA (Generally Available), it’s not yet fully production ready.
+We want to share it with the world now and get the invaluable feedback from you - Apiary users.
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
+in ApiaryUI. Some requested features not present in v3 were implemented to ApiaryUI.
+We’ve managed to fix tons of v3 bugs inside the new ApiaryUI. 
+
+As the architecture of ApiaryUI is very simple and solid, we are able to add new features
+and fix bugs in very rapid pace. Our extensive suite of unit, integration and e2e tests
+are helping with that. This can finally release further evolution of [API Blueprint](https://apiblueprint.org/) language.
```

Comment:
I wouldn't disclose the last sentence. While true, our customers are not aware and this makes us either a laughing stock and/or a very particular promise. I wouldn't promise, I'd deliver. 

Let's start the APIB feature announcement blog post with "Thanks to our new documentation, we've been able to release...".

Reply:
Removed